src/sagemaker/session.py

```diff
@@ -2349,9 +2366,15 @@ def logs_for_auto_ml_job(  # noqa: C901 - suppress complexity warning for this m
 
         description = _wait_until(lambda: self.describe_auto_ml_job(job_name), poll)
 
-        instance_count, stream_names, positions, client, log_group, dot, color_wrap = _logs_init(
-            self.boto_session, description, job="AutoML"
-        )
+        (
+            instance_count,
+            stream_names,
+            positions,
+            client,
+            log_group,
+            dot,
+            color_wrap,
+        ) = _logs_init(self.boto_session, description, job="AutoML")
```

Comment:
Is there a reason we are moving every single param to next line? Is it a best practice for Python code development? 
If not, let's remove this formatting since it will just increase the number of lines in this huge file(~7K lines). AFAIK, we have a line length limit of 100, if it satisfies that, lets keep it as original.
If there are any black-check or pylint errors we need to resolve, we can move them to next line.

Reply:
Thanks I believe this was the result of the ``black-format`` command to auto fix the file,  I can double check and revert this